app/assets/javascripts/mobile.js

```diff
@@ -30,11 +25,13 @@ $(function () {
         $this.text($this.text().replace(/Show/, 'Hide'));
         instance.$navbar.removeClass('page-navigation-closed');
         instance.$navbar.addClass('page-navigation-open');
+        GOVUK.analytics.trackEvent("contentsClicked", "openGuideNav",{});
```

Comment:
This needs to be guarded like https://github.com/alphagov/static/blob/master/app/assets/javascripts/modules/track-click.js#L43 if the analytics module is not loaded